<launch>
	<node name="resave_marker_pose_client" pkg="resave_marker_pose" type="resave_marker_pose_client" respawn="false" output="screen">
	</node>
</launch>
